apps/insights/src/components/PriceFeeds/index.tsx

```diff
@@ -124,6 +124,29 @@ export const PriceFeeds = async () => {
           />
         </UnstyledTabPanel>
       </UnstyledTabs>
+      <Card
+        className={styles.trademarkDisclaimerCard}
+        variant="tertiary"
+        title="Trademark Disclaimer"
+      >
+        <div className={styles.trademarkDisclaimerContent}>
+          <p>
+            The Pyth Network name, logo, and all related names, logos, product
+            and service names, designs, and slogans are trademarks of the Pyth
+            Data Association or its affiliates or licensors.
```

Comment:
Please replace this text with the full contents of https://www.pyth.network/trademark-disclaimer